recipes/easy_profiler/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, tools, CMake
+import os
+import glob
+
+
+class easy_profilerConan(ConanFile):
```

Comment:
```suggestion
class EasyProfilerConan(ConanFile):
```

https://www.python.org/dev/peps/pep-0008/#id41